examples/pytorch/chatglm6b/finetune.py

```diff
@@ -228,7 +243,10 @@ def cfg_modify_fn(cfg):
         rank=args.lora_rank,
         lora_alpha=args.lora_alpha,
         lora_dropout=args.lora_dropout)
-    model = model.bfloat16()
+    if args.use_amp:
+        model = model.float()
```

Comment:
will changing model to float  affect the accuracy of the finetuned model?

Reply:
amp will do the mixed precision thing, so if args.use_amp is True, the amp hook will be added